substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationParser.java

```diff
@@ -28,15 +28,19 @@
 import java.io.Reader;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 import com.oracle.svm.core.util.json.JSONParserException;
 
 public abstract class ConfigurationParser {
+    private final Map<String, Set<String>> seenUnknownAttributes = new HashMap<>();
```

Comment:
This should be named `seenUnknownAttributesByType` to be more descriptive.

Reply:
I agree, it makes more sense to name it `seenUnknownAttributesByType` - fixed